### PR TITLE
WINC support notice and feature name update

### PR DESCRIPTION
**Pull request by codyhoag**
Addressing support issues brought up in openshift/windows-machine-config-operator#226. The feature name has also been updated.

### CODE REVIEW

**Comment by codyhoag** [comment]
@Anandnatraj PTAL

**Comment by codyhoag** [comment]
These changes have been ACKed by Product Management via email.

File: windows_containers/windows-containers-release-notes.adoc [hunk 1]
```diff
@@ -1,17 +1,22 @@
...
+Windows Container Support for Red Hat OpenShift is a feature providing the ability to run Windows compute nodes in an {product-title} cluster. This is possible by using the Red Hat Windows Machine Config Operator (WMCO) to install and manage Windows nodes. With Windows nodes available, you can run Windows container workloads in {product-title}.
 
 The release notes for Red Hat OpenShift for Windows Containers tracks the development of the WMCO, which provides all Windows container workload capabilities in {product-title}.
 
+[id="getting-support"]
+== Getting support
+
+You must have a subscription to receive support for the Red Hat WMCO. Deploying Windows container workloads in production clusters is not supported without a subscription. If you do not have a subscription, you can use the community WMCO, a distribution that lacks official support. Support should be requested through the link:http://access.redhat.com/[Red Hat Customer Portal].
```
Comment (kalexand-rh) [inline windows_containers/windows-containers-release-notes.adoc:18]: > ! should v
```suggestion
You must have a subscription to receive support for the Red Hat WMCO. Deploying Windows container workloads in production clusters is not supported without a subscription. If you do not have a subscription, you can use the community WMCO, a distribution that lacks official support. Request support through the link:http://access.redhat.com/[Red Hat Customer Portal].
```
Always prefer a more direct verb or way of saying something. Omit “should,” or use “must” or “might,” as appropriate.

**Review by kalexand-rh — COMMENTED** [review]
Aside from one style pick, this LGTM!

**Comment by codyhoag** [comment]
@vikram-redhat @sjstout this aligns docs with the WINC updates necessary for the GA launch.

**Pull request merged**